src/py42/sdk/queries/fileevents/filters/activity_filter.py

```diff
@@ -0,0 +1,13 @@
+from py42.sdk.queries.query_filter import QueryFilterBooleanField
+
+
+class TrustedActivity(QueryFilterBooleanField):
+    """Class that filters events based on whether activity can be trusted."""
+
+    _term = u"trusted"
+
+
+class RemoteActivity(QueryFilterBooleanField):
+    """Class that filters events based on whether the remote activity can be trusted."""
```

Comment:
This is just based on whether or not the activity was remote (took place outside of corporate IP range), you can remove the part about being trusted.

Reply:
Changed.